app/views/admin/users/_form.html.erb

```diff
@@ -48,7 +48,13 @@
                     <% end %>
                 </div>
             </div>
-
+            <h3 class="p3"><%= t('users.form.newsletters') %></h3>
+            <div class="field">
+                <div class="ui checkbox">
+                    <%= f.check_box :auction_newsletter, { value: user.auction_newsletter }, "true", "false" %>
```

Comment:
Value is taken from model automatically. There is no need to explicitly provide it.

Reply:
What these `true` and `false` mean?

Reply:
Mhm, both removed. By syntax they're meant to define output of checkbox (1 or 0, true or false etc), but not really needed in this scenario. As other checkboxes (tos, for example) were implemented same way, I decided to follow this pattern.